install_config/topics/manually_configuring_for_gce-proc.adoc

```diff
@@ -0,0 +1,162 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+= Manually configuring {product-title} for GCE
+
+== Manually configuring master hosts for GCE
+
+Perform the following on all master hosts.
+
+.Procedure
+
+. Edit the master configuration file at `/etc/origin/master/master-config.yaml` by
+default on all masters and update the contents of the `apiServerArguments` and
+`controllerArguments` sections:
++
+[source,yaml]
+----
+kubernetesMasterConfig:
+  ...
+  apiServerArguments:
+    cloud-provider:
+      - "gce"
+    cloud-config:
+      - "/etc/origin/cloudprovider/gce.conf"*
+  controllerArguments:
+    cloud-provider:
+      - "gce"
+    cloud-config:
+      - "/etc/origin/cloudprovider/gce.conf"*
+----
++
+[IMPORTANT]
+====
+When triggering a containerized installation, only the *_/etc/origin_* and
```

Comment:
There's no `containerized installation` concept for ocp-3.10, master always run as static pod and node could run as systemd services or system container.